index.html

```diff
@@ -3,20 +3,22 @@
 <head>
     <meta charset="UTF-8">
     <link href="css/style.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/modal.css">
     <link href='https://fonts.googleapis.com/css?family=Playfair+Display' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://use.fontawesome.com/75d401af0c.js"></script>
     <title>Felix Odete's Portfolio</title>
     <meta content="width=device-width, initial-scale=1" name="viewport">
 </head>
 <body>
```

Comment:
- [ ] Kindly fix the website desktop version responsively as some part of the page are overflowing
<img width="1440" alt="Screen Shot 2022-04-15 at 11 50 05" src="https://user-images.githubusercontent.com/28307960/163556614-e14cb074-ea80-4e06-8a33-1e9934eb507e.png">

Reply:
Thanks for the observation. Looked into it and fixed it.

<img width="1440" alt="Screenshot 2022-04-15 at 14 40 02" src="https://user-images.githubusercontent.com/40231363/163566615-aa31618f-3ab1-4005-9c57-11869555ef1a.png">